### DISCUSSION TITLE
UR bar timing indicator should be a turtle

### DISCUSSION THREAD

**Discussion by AosUK** [ppy/osu#25952] [Ideas] [answered]
Rather than a bike and a faster bike, perhaps a bike and a turtle. As it is more clear what side is early and late.

**Comment by cloudiees** [comment]
Personally I feel like the current icons for the UR bar are a little unintuitive. For me personally I find a truck (the current icon for early/too fast) as slow. While I find a bike (current icon for late/too slow) as fast. This is probably because I intuitively compare a truck to a car (most of the time a truck is slower than a car) and a bike to walking (most of the time biking is faster than walking), and seeing these two objects together my conflicting innate comparisons kinda makes my brain shart itself and makes me confused. idk I might be the only one who sees it like this.

ok paragraph over, suggestion: maybe turn the late/too slow icon into something more generic that most players dont have to think about such as a snail (or a turtle as suggested), and change the early/too fast to a fast animal that is easy to depict in a simplistic style (kinda hard and unspecific ik), or just keep it as some vehicle. Personally it would clear things up for me a lot more

**Comment by Lyti4n** [comment, marked as answer]
This is fixed https://github.com/ppy/osu/pull/27787
